packages/replayer/src/replayer.ts

```diff
@@ -31,27 +31,26 @@ export const replayEvents: ReplayEventsFn = async (options) => {
     outputDir,
     session,
     sessionData,
+    replayExecutionOptions,
+    dependencies,
+  } = options;
+
+  // Extract replay metadata
+  const {
     headless,
     devTools,
-    dependencies,
-    padTime,
     shiftTime,
     networkStubbing,
     accelerate,
-  } = options;
-
-  // Extract replay metadata
+    padTime,
+  } = replayExecutionOptions;
   const metadata: ReplayMetadata = {
     session,
     options: {
       appUrl,
       outputDir,
-      headless,
-      devTools,
       dependencies,
-      padTime,
-      shiftTime,
-      networkStubbing,
+      ...replayExecutionOptions,
```

Comment:
Note that previously this was missing some of the options we added recently, like acceleration. Going forward that shouldn't be a risk since they'll be added automatically (which in this case is what we presumably want - just capture all the options).